Update Arabic.html

AbdallahQashmar opened the pull request:
The code is still not ready, i'll edit the links and the size of the pictures

ysmeen24 committed:
Update WelcomePage.html

I updated the welcome page please review.

ysmeen24 committed:
Merge pull request #30 from ysmeen24/WelcomePage-1

Update WelcomePage.html

ysmeen24 committed:
Create README.md

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

AbdallahQashmar committed:
Update Arabic.html

The code is still not ready, i'll edit the links and the size of the pictures

ysmeen24 commented on WelcomePage/SignAndLog/Homepage/Cuisines/Arabic.html:
```diff
@@ -3,8 +3,52 @@
+		<h1>Al Hallab</h1>
+		<a href="https://alhallabrestaurant.com/"><img src="AlHallab.jpg" alt="Logo" width="131" height="131"></a>
+		<h4>Rating:</h4>
+		<img src="4stars.png" alt="4 stars" width="112" height="30">
```
Great Job Abdullah. However make sure to link the images from the repo images folder for example (repo link /project/Images/ImageFile.). Thank you.

ysmeen24 reviewed (changes requested):
Great job Abdullah but make sure to link the images from the repo. Thank you.

AbdallahQashmar committed:
Update Arabic.html

The Arabic Cuisine is done, Check it and let me know if there's something wrong

ysmeen24 commented:
Great job everything works properly now.
